guides/manual-installation.md

```diff
@@ -514,18 +518,79 @@ docker run -d \
 
 ...[other settings]...
 
-# REQUIRED:
+# Required:
 -e ENABLE_OIDC=true  \
 -e OIDC_CLIENT_ID=<<OIDC_CLIENT_ID>> \
 -e OIDC_CLIENT_SECRET=<<OIDC_CLIENT_SECRET>> \
 -e OIDC_BASE_URL=<<OIDC_BASE_URL>> \
 
-# OPTIONAL:
+# Optional:
 -e OIDC_CALLBACK_URL=<<OIDC_CALLBACK_URL>> \
 
 ...[other settings]...
 ```
 
+### OAuth 2.0
+
+Trento integrates with an IDP that uses the OAuth 2 protocol to authenticate users accessing the Trento web console. Authorization for specific abilities/permissions is managed by Trento, which means that only basic user information is retrieved from the external IDP.
+
+#### User Roles and Authentication for OAuth 2.0
+The authentication process is the same as in [User Roles and Authentication for OIDC](#user-roles-and-authentication-for-oidc).
+
+#### Enabling OAuth 2.0
+
+OAuth 2.0 authentication is **disabled by default**.
+
+##### Enabling OAuth 2.0 when using RPM packages
+
+Provide the following environment variables to trento-web configuration, which is stored at ```/etc/trento/trento-web```  and restart the application to enable OIDC integration.
+
+```
+# Required:
+ENABLE_OAUTH2=true
+OAUTH2_CLIENT_ID=<<OAUTH2_CLIENT_ID>>
+OAUTH2_CLIENT_SECRET=<<OAUTH2_CLIENT_SECRET>>
+OAUTH2_BASE_URL=<<OAUTH2_BASE_URL>>
+OAUTH2_AUTHORIZE_URL=<<OAUTH2_AUTHORIZE_URL>>
+OAUTH2_TOKEN_URL=<<OAUTH2_TOKEN_URL>>
+OAUTH2_USER_URL=<<OAUTH2_USER_URL>>
+
+# Optional:
+OAUTH2_SCOPES=<<OAUTH2_SCOPES>>
+OAUTH2_CALLBACK_URL=<<OAUTH2_CALLBACK_URL>>
+```
+
+> **Note:** OAUTH2_SCOPES is an optional variable with the default value ```openid profile email```. OAUTH2_SCOPES must be adjusted depending on IDP provider requirements.
```

Comment:
Let's remove this `Note`. With the one below, after the docker instructions is enough I would say.
What do you think?